src/vs/editor/browser/viewParts/indentGuides/indentGuides.ts

```diff
@@ -160,13 +156,13 @@ export class IndentGuidesOverlay extends DynamicViewOverlay {
 	private getGuidesByLine(
 		visibleStartLineNumber: number,
 		visibleEndLineNumber: number,
-		activeCursorPosition: Position | null
+		activeCursorPositions: Position[]
 	): IndentGuide[][] {
 		const bracketGuides = this._bracketPairGuideOptions.bracketPairs !== false
 			? this._context.viewModel.getBracketGuidesInRangeByLine(
 				visibleStartLineNumber,
 				visibleEndLineNumber,
-				activeCursorPosition,
+				activeCursorPositions[0],
```

Comment:
This is problematic. I think here you would have to pass in all the active positions.